src/mono/wasm/runtime/scheduling.ts

```diff
@@ -48,14 +49,18 @@ function mono_background_exec_until_done() {
 
 export function schedule_background_exec(): void {
     ++pump_count;
-    setTimeout(mono_background_exec_until_done, 0);
+    Module.safeSetTimeout(mono_background_exec_until_done, 0);
 }
 
 let lastScheduledTimeoutId: any = undefined;
 export function mono_wasm_schedule_timer(shortestDueTimeMs: number): void {
     if (lastScheduledTimeoutId) {
         clearTimeout(lastScheduledTimeoutId);
         lastScheduledTimeoutId = undefined;
+        Module.runtimeKeepalivePop();
```

Comment:
where's the runtimeKeepalivePop that matches this?

Reply:
`Module.safeSetTimeout` does the `runtimeKeepalivePush` but clearTimeout is asymmetric. Added comment